Tasks/SonarQubePostTest/SonarQubeMetrics.ps1

```diff
@@ -38,18 +38,38 @@ function WaitForAnalysisToFinish
     Assert (![String]::IsNullOrEmpty($analysisId)) "Could not fetch the analysis id"
 }
 
+
+function FetchQualityGateDetails
+{
+    param ([Array]$errors,[Array]$warnings)
+    
+    $analysisId = GetTaskContextVariable "MSBuild.SonarQube.AnalysisId"             
+    Assert (![String]::IsNullOrEmpty($analysisId)) "WaitForAnalysisToFinish should be called first."
+       
+    $response = InvokeGetRestMethod "/api/qualitygates/project_status?analysisId=$analysisId" $true
```

Comment:
How are different versions of the SQ server being handled (specifically older versions that don't have this API)?
Is there a minimum level of SQ server that is required for the quality gate summary to appear?